app/scienceapi/events/models.py

```diff
@@ -70,15 +72,26 @@ class Event(models.Model):
         help_text='Slug appended to the Mozilla Science event\'s '
                   'url that represents this event',
     )
+    is_virtual = models.BooleanField(
+        default=False,
+        help_text='When selected, events will appear on site converted'
+                  'to a visitor\'s local time.',
+        verbose_name='Virtual Event'
+    )
+
+    ALL_TIMEZONES = [(i, i) for i in pytz.all_timezones]
+    timezone = models.CharField(
+        max_length=40,
+        choices=ALL_TIMEZONES,
```

Comment:
You have basically just changed the field by adding choices. As I mention, move that validation to the form instead. Either way, one consequence is that you will need to figure out how to migrate all the existing data to this new format.